content/blog/setting-up-harbor-as-a-local-container-registry-in-hpe-private-cloud-ai.md

```diff
@@ -0,0 +1,304 @@
+
+### Pushing images to Harbor registry
+
+With the project and users set up, you're ready to push the container images to *Harbor* by following these steps:
+
+* *Log in to Harbor registry*
+
+Use the Docker client to authenticate with the *Harbor* registry using the *pcai-admin* user credentials, by running the following command :
+
+```shell
+$ docker login harbor.ingress.pcai0104.ld7.hpecolo.net
+Username: pcai-admin
+Password:
+
+WARNING! Your credentials are stored unencrypted in '/home/guoping/.docker/config.json'.
+Configure a credential helper to remove this warning. See
+https://docs.docker.com/go/credential-store/
+
+Login Succeeded
+```
+
+If you get any certificate error when logging in from a Linux client, update the file */etc/docker/daemon.json* by adding the following entry, replacing the *Harbor* registry URL with your own: 
+
+```shell
+{
+  "insecure-registries" : [" harbor.ingress.pcai0104.ld7.hpecolo.net "]
+}
+```
+
+After making this change, reload the daemon and restart the Docker service:
+
+```shell
+$ sudo systemctl daemon-reload
+$ sudo systemctl restart docker
+```
+
+* *Tag an existing image*
+
+Rather than building a Docker image from a Dockerfile, we'll pull the sample CFE Nginx image, *'pcaidemo/cfe-nginx'*, from *DockerHub* and tag it with the *Harbor* registry URL and project name: 
+
+```shell
+$ docker images
+REPOSITORY           TAG       IMAGE ID       CREATED        SIZE
+pcaidemo/cfe-nginx   v0.1.0    1e5f3c5b981a   2 months ago   192MB
+
+$ docker tag pcaidemo/cfe-nginx:v0.1.0 harbor.ingress.pcai0104.ld7.hpecolo.net/demo/cfe-nginx:v0.1.0
+
+$ docker images
+REPOSITORY                                               TAG       IMAGE ID       CREATED        SIZE
+pcaidemo/cfe-nginx                                       v0.1.0    1e5f3c5b981a   2 months ago   192MB
+harbor.ingress.pcai0104.ld7.hpecolo.net/demo/cfe-nginx   v0.1.0    1e5f3c5b981a   2 months ago   192MB
+```
+
+* *Push the image to Harbor registry*
+
+Push the image to the *Harbor* registry by running the following command:
+
+```shell
+$ docker push harbor.ingress.pcai0104.ld7.hpecolo.net/demo/cfe-nginx:v0.1.0
+The push refers to repository [harbor.ingress.pcai0104.ld7.hpecolo.net/demo/cfe-nginx]
+7e893c1b6ee8: Pushed
+463308bed0c9: Pushed
+4197a611afec: Pushed
+3e96162769d5: Pushed
+892e805f6f4f: Pushed
+626ab8a5d57b: Pushed
+7fb72a7d1a8e: Pushed
+v0.1.0: digest: sha256:114dff0fc8ee3d0200c3a12c60e3e2b79d0920dd953175ecb78a0b157425b25e size: 1778
+```
+
+* *Verify the image from Harbor registry* 
+
+From the *Harbor* UI, the image *cfe-nginx* appears under *Repositories* tab of the *demo* project:
+
+![](/img/demo-project.png)
+
+From  the Docker client, log in to the *Harbor* registry as the *pcai-developer* user, then pull the image *cfe-nginx* from the registry. The image downloads successfully, confirming that the user has appropriate access and the *Harbor* registry is functioning as expected.
```

Comment:
You might want to consider rephrasing the second sentence to "Hopefully the image downloads successfully, confirming that the user..."

OR, "The image should download successfully, confirming that the user..."